Makefile

```diff
@@ -46,39 +48,10 @@ UI_SERVICE_NAME := $(or $(UI_SERVICE_NAME),assisted-installer-ui)
 # Monitoring services
 PROMETHEUS_SERVICE_NAME := $(or $(PROMETHEUS_SERVICE_NAME),prometheus-k8s)
 
-# nodes params
-ISO := $(or $(ISO), "") # ISO should point to a file that has the '.iso' extension. Otherwise deploy will fail!
-NUM_MASTERS :=  $(or $(NUM_MASTERS),3)
-WORKER_MEMORY ?= 8892
-MASTER_MEMORY ?= 16984
-NUM_WORKERS := $(or $(NUM_WORKERS),0)
-NUM_DAY2_WORKERS := $(or $(NUM_DAY2_WORKERS),1)
-STORAGE_POOL_PATH := $(or $(STORAGE_POOL_PATH), $(PWD)/storage_pool)
-CLUSTER_ID := $(or $(CLUSTER_ID), "")
-CLUSTER_NAME := $(or $(CLUSTER_NAME),test-infra-cluster)
-REMOTE_SERVICE_URL := $(or $(REMOTE_SERVICE_URL), "")
-WORKER_DISK ?= 21474836480
-MASTER_DISK ?= 128849018880
-DISABLED_HOST_VALIDATIONS := $(or $(DISABLED_HOST_VALIDATIONS),"")
-
-#hyperthreading
-HYPERTHREADING := $(or $(HYPERTHREADING), all)
-
 # network params
 NAMESPACE := $(or $(NAMESPACE),assisted-installer)
 BASE_DNS_DOMAINS := $(or $(BASE_DNS_DOMAINS), "")
 BASE_DOMAIN := $(or $(BASE_DOMAIN),redhat.com)
```

Comment:
what about this one?

Reply:
We are using it on `assisted_deployment.sh` so I don't think it can be done (unless we change the service deployment from bash script to python script)